instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
Also re 
> I don't feel the permission really belongs on a model, since it controls access to more than just that model.

True, but everything it does control access to is either that model or conceptually a "child" of that model.

InstanceReferences own concrete Instance subclasses which own concrete AppServer subclasses which own concrete Server subclasses, and all of those owned things also own log entries. The permission controls access to all those things and their APIs.

Overall I'm fine with any of these approaches. If you want my opinion, I do prefer @smarnach 's code over the one taken in the PR (if you can confirm that django doesn't think the content type is 'stale' upon subsequent migrations and prompt the user to delete it), and I prefer the InstanceReference approach overall though.